Upgraded to storybook 7.3.2

juanparati opened the pull request:
I upgrade to storybook 7.3.2. I also fix the issue with SSL validation.

zyc9012 commented on .storybook/main.js:
```diff
@@ -1,12 +1,26 @@
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
+    getAbsolutePath("@storybook/addon-links"),
```
I removed all the `getAbsolutePath` wraps and it still worked. Why do we need it?

zyc9012 commented on babel.config.json:
```diff
@@ -0,0 +1,17 @@
+        }
+      }
+    ],
+	"@babel/preset-flow",
```
Let's format this file

zyc9012 commented on packages/core/src/modules/cell.ts:
```diff
@@ -153,10 +153,6 @@ export function setCellValue(
         delete cell.f;
       }
 
-      // if (!_.isNil(v.spl)) {
```
Let's not edit files unrelated to this PR

zyc9012 reviewed (changes requested):
Thanks @juanparati for upgrading storybook 👍.

Let's also fix type deprecations in story files, something like this

```diff
diff --git a/stories/API.stories.tsx b/stories/API.stories.tsx
index a54c1aa..4a1a3b9 100644
--- a/stories/API.stories.tsx
+++ b/stories/API.stories.tsx
@@ -1,11 +1,11 @@
 import React, { useCallback, useRef, useState } from "react";
-import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Meta, StoryFn } from "@storybook/react";
 import { Workbook, WorkbookInstance } from "@fortune-sheet/react";
 import { Sheet } from "@fortune-sheet/core";
 
 export default {
   component: Workbook,
-} as ComponentMeta<typeof Workbook>;
+} as Meta<typeof Workbook>;
 
 const ApiExecContainer: React.FC<{
   onRun: () => any;
@@ -43,7 +43,7 @@ const ApiExecContainer: React.FC<{
   );
 };
 
-export const GetCellValue: ComponentStory<typeof Workbook> = () => {
+export const GetCellValue: StoryFn<typeof Workbook> = () => {
   const ref = useRef<WorkbookInstance>(null);
   const [data, setData] = useState<Sheet[]>([
     {
```

----

The default pages becomes "Docs", which is weird, could you fix that?

![image](https://github.com/ruilisi/fortune-sheet/assets/3034310/5dd7d7bf-c9b1-4f41-9d6c-4f8816b8fb58)

juanparati commented:
@zyc9012: All changes are done:
- Reverted unrelated [28bacdb](https://github.com/ruilisi/fortune-sheet/pull/470/commits/28bacdb41ce18e9c2a28f5385ce9090ea002cdc7) and [b775c6d](https://github.com/ruilisi/fortune-sheet/pull/470/commits/b775c6dd4b209263d51d25754bf9a27b90cbad44)
- Replaced deprecated types
- Removed Docs link from storybook
- Removed "getAbsolutePath" method
- Reformatted babel.config.json

zyc9012 commented on packages/core/src/modules/cell.ts:
```diff
@@ -143,6 +143,10 @@ export function setCellValue(
 
   let vupdate;
 
+  if (!_.isNil(v.v) && _.includes(v.ct?.fa, "0.0")) {
```
Let's not introduce unrelated code here.

zyc9012 reviewed (changes requested):
Thanks @juanparati 

Let's also update `Collabration.stories.tsx` and `Features.stories.tsx`

juanparati commented:
@zyc9012 : Changes made:
- Removed unrelated
- Replaced deprecated types for stories
- Replaced autodocs to "tag"

zyc9012 reviewed (approved):
Thanks @juanparati 👍
